staging/src/k8s.io/kube-gen/cmd/client-gen/generators/generator_for_clientset.go

```diff
@@ -69,8 +69,15 @@ func (g *genClientset) GenerateType(c *generator.Context, t *types.Type, w io.Wr
 	// perhaps we can adapt the go2ild framework to this kind of usage.
 	sw := generator.NewSnippetWriter(w, c, "$", "$")
 
-	allGroups := clientgentypes.ToGroupVersionPackages(g.groups)
+	type groupArg struct {
```

Comment:
Could you just add `LowerCaseGroupVersion` to `GroupVersionPackage` and update `ToGroupVersionPackages`